extension-manifest-v2/src/background.js

```diff
@@ -129,6 +131,8 @@ function tryOpenOnboarding() {
 		return;
 	}
 
+	conf.enable_autoconsent = false;
```

Comment:
See comment in ConfData. If it would start with false and gets enabled at the end of the onboarding, this might not be needed (like for adblocking and anti-tracking).

Reply:
See my comment above. We have to cover a case of user, which updates with onboarding already completed.